NDB.Covid19/NDB.Covid19/ViewModels/QuestionnaireViewModel.cs

```diff
@@ -132,6 +132,10 @@ public void SetSelection(QuestionaireSelection selection)
                 }
                 PersonalData.FinalMiBaDate = mibaDate < _localSelectedDate ? mibaDate : _localSelectedDate;
             }
+            else
+            {
+                PersonalData.FinalMiBaDate = mibaDate;
```

Comment:
I think maybe we should rename it to MSIS date instead?

Reply:
Yes, makes sense